### ISSUE TITLE
Yugabyet guarentess against data loss

### ISSUE THREAD

**Issue by amit2103** [YugaByte/yugabyte-db#355]
Guys,

We want to do cross region replication with Yubabyte.
So assuming we have an instance in Tokyo and another in China..We see that some time tramsmission fails for 20% packets (ping). If Yugabyte tries to transfer data, is it able to handle packet loss ?

**Comment by rkarthik007** [comment]
Hi @amit2103,

Yes, YugaByte DB can handle packet loss since it relies on TCP internally for communication. In fact, we had a discussion with Kyle (Jepsen) about implementing packet loss failure scenario tests and he pointed out that this is not practically useful if we use TCP. @ttyusupov - please correct me if I am wrong.

**Comment by ttyusupov** [comment]
Hi @rkarthik007 @amit2103, yes, you can find answer from Kyle there - https://github.com/jepsen-io/jepsen/issues/254

**Comment by amit2103** [comment]
Cool . Is Yugabyte now Jespen test proof ?

**Comment by rkarthik007** [comment]
Yes, we have been running the tests repeatedly. We have implemented most faults and have verified a lot of the tests successfully. We are close to completing this effort.

**Comment by amit2103** [comment]
Thanks for the help guys thats all from my side

**Comment by rkarthik007** [comment]
Most welcome! Closing this issue, please feel free to reopen/open another one.
